.github/workflows/test.yml

```
@@ -166,12 +166,35 @@ jobs:
             result/bin/crio-status
             result/bin/pinns
 
+  build-static-ppc64le:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: cachix/install-nix-action@v20
+        with:
+          install_url: https://releases.nixos.org/nix/nix-${{ env.NIX_VERSION }}/install
+      - uses: cachix/cachix-action@v12
+        with:
+          name: cri-o-static-ppc64le
```

Ah my fault, let's revert back to the existing cachix cache:
```suggestion
          name: cri-o-static
```

---

Hi @saschagrunert,
Reverted the changes as suggested.